modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1340,6 +1340,11 @@
      */
     public static final String IGNITE_PAGES_LIST_DISABLE_ONHEAP_CACHING = "IGNITE_PAGES_LIST_DISABLE_ONHEAP_CACHING";
 
+    /**
+     * Disable group state lazy store. Should be used for test purposes only.
```

Comment:
Please add more descriptive comment on how does this property affect the system's behavior.